### PR TITLE
Update VMware library installation docs

### PR DESCRIPTION
**Pull request by jakelevinez**
##### SUMMARY
changed pip to pip3 for usable installation commands

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
changed pip to pip3 so installation command actually works

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
none

##### ADDITIONAL INFORMATION
none

```

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_intro.rst [hunk 1]
```diff
@@ -21,15 +21,15 @@ and vCenter infrastructure. You can install pyVmomi using pip:
 
 .. code-block:: bash
 
-    $ pip install pyvmomi
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_intro.rst]: Add a note that you can use pip as well.

Comment (samccann) [reply docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_intro.rst]: I'm curious why this is necessary (changing pip to pip3) - doesn't it depend on the distro? For example, I'm running Fedora 32 and it isn't called pip3... it's still just pip (though running python 3.8).

File: docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_intro.rst [hunk 2]
```diff
@@ -21,15 +21,15 @@ and vCenter infrastructure. You can install pyVmomi using pip:
...
+    $ pip3 install pyvmomi
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_intro.rst]: Thanks @jakelevinez I'd prefer to leave the code example using `pip` but update the line above to say:
"You can install pyVmomi using pip (you may need to use `pip3`, depending on your OS/distro):"

**Comment by Akasurde** [comment]
@jakelevinez Thanks for the contribution. @acozine @samccann  Thanks for the review.

**Pull request merged**